contracts/errors/ServiceAgreementErrorsV1.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.4;
 
 library ServiceAgreementErrorsV1 {
     error EmptyAssetCreatorAddress();
-    error AssetStorageNotInTheHub(address contractAddress);
+    error AssetStorgeNotInTheHub(address contractAddress);
```

Comment:
So is this intended typo so it takes less storage?

Reply:
No, I just missed the typo :)